pkg/cmd/pkg/package_describe_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TesPkgDescribeShouldProxyCommandsToCosmos(t *testing.T) {
+func TestPkgDescribeShouldProxyCommandsToCosmos(t *testing.T) {
```

Comment:
This is why we should run liners on `_test.go`. This typo prevents this test from run. So we will never see it failing.

Reply:
Which linter checks this?